package.json

```diff
@@ -22,7 +22,7 @@
     "@bcrs-shared-components/base-address": "2.0.3",
     "@bcrs-shared-components/breadcrumb": "2.1.15",
     "@bcrs-shared-components/business-lookup": "1.3.4",
-    "@bcrs-shared-components/certify": "2.1.51",
+    "@bcrs-shared-components/certify": "2.1.55",
     "@bcrs-shared-components/completing-party": "2.1.30",
```

Comment:
The current dev correctly display the required wording ("are" authorized). Maybe we don't need to update it to 2.1.55. 
But when I run locally, it displays "is" instead of "are".

Could you please suggest, thanks.

Reply:
Check your node_modules folder. You might have an older version than 51.

As for whether to upgrade from 51 to 55...
- on the one hand, it's good to keep these imports updated
- on the other hand, did very much change from 51 to 55? if so then extra testing is needed

If there are no significant changes from 51 to 55 then I leave it up to you whether to upgrade it.

Reply:
It seems there are no changes from 51 to 55. They were updated just due to the chore(release) publish
![image](https://github.com/user-attachments/assets/95993dd4-9f5e-4dea-811a-b99677e427d4)

Reply:
That likely means some dependency was updated (eg, enums or interfaces).

Up to you whether to update to 55 or not.

Reply:
So I leave it upgraded